### DISCUSSION TITLE
Particle dissolution model in Spatial Structure instead of Events?

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1127] [Questions & Problems] [answered]
I want to recreate the particle dissolution model for a custom MoBi model but have the solid particles contained within a single compartment. Since the solid particles are staying in a single compartment, is there any benefit to having the particle dissolution model within the Events building block? Or is it better to recreate the particle dissolution "module" within the Spatial Structure? My understanding is that the main benefit of having the original particle dissolution model within the Events building block is you can track the transit of solid particles through the intestinal tract, but that is not needed for my model. Does anyone have any thoughts on this?

**Comment by Yuri05** [comment, marked as answer]
The main reason is in fact the Building Block principle: something like drug amount, particles radius distribution, number of particles etc. are properties of formulation/administration, not of a spatial structure.
In MoBi formulation/administration properties are stored in Events building block (which is in fact **Events and Applications** building block). Actually the Events building Block in MoBi is implemented not in the optimal way, e.g. separating formulations into their own Building block (like in PK-Sim) would be much better.

Technically you could place the dissolution into the spatial structure (e.g. by the definiton of a "solid drug"-subcontainer for each administration and each particle size). The effort would be even higher in this case (e.g. you need to define the neighborhoods between those subcontainers and your dissolution compartment).
And if you would use this spatial structure with another dosage/formulation form: it would contain many unused containers/neighborhoods.

But let me ask you the other way around: what are the disatvantages/problems of defining the particles dissolution in a single compartment via the Events/Applications Building Block?

**Reply by moPellowe** [reply]
Thank you for explaining the Building Block principle. I missed that the drug formulation properties were meant to be part of the Events building block. Also, I see your point about having unused containers/neighborhoods when changing formulations.

To be honest, I didn't see any disadvantage to having the particle dissolution in the Events building block. But at the time, I also didn't see any disadvantage to having the particle dissolution model in the Spatial Structure (but you make a good point about changing formulations). I just wanted to know if there was a good reason why it was in one building block vs another, and it seems there is one. So thanks, @Yuri05 !
